flake.nix

```diff
@@ -29,18 +30,35 @@
       overlay = final: prev: {
 
         pcb-rnd = with final; stdenv.mkDerivation rec {
-          name = "pcb-rnd-${version}";
+          pname = "pcb-rnd";
+
+          inherit version;
 
           src = pcb-rnd-src;
 
-          buildInputs = [ ];
+          buildInputs = [ final.librnd ];
```

Comment:
just a question, do you need it as a buildInputs if you have it as a prefix below ? (just wondering)

Reply:
No it's not necessary.

Reply:
But I think it's nice to specify it anyways. It makes the dependency more obvious.

Reply:
cool, I don't have strong feelings about it.